server-openid4vci/src/main/java/com/android/identity/server/openid4vci/QrServlet.kt

```diff
@@ -0,0 +1,48 @@
+package com.android.identity.server.openid4vci
+
+import com.google.zxing.qrcode.decoder.ErrorCorrectionLevel
+import com.google.zxing.qrcode.encoder.Encoder
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import java.awt.image.BufferedImage
+import java.awt.image.DataBufferByte
+import javax.imageio.ImageIO
+
+
+/**
+ * Servlet that encodes text string (passed as `q` parameter) into a QR code.
+ */
+class QrServlet : BaseServlet() {
+    override fun doGet(req: HttpServletRequest, resp: HttpServletResponse) {
+        val str = req.getParameter("q") ?: ""
```

Comment:
Do we want to do any sanity checking on the string? Check that it's not 0-length, at least? Or will Encoder.encode already do that?